rust/nasl-builtin-cryptographic/tests/helper/mod.rs

```diff
@@ -0,0 +1,7 @@
+use std::num::ParseIntError;
+pub(crate) fn decode_hex(s: &str) -> Result<Vec<u8>, ParseIntError> {
+    (0..s.len())
+        .step_by(2)
+        .map(|i| u8::from_str_radix(&s[i..i + 2], 16))
+        .collect()
+}
```

Comment:
This code is [duplicated](https://github.com/greenbone/openvas-scanner/pull/1431/files#diff-0cb1d2de0b3359b2737a9099ecd00899a89342c6a55b484626f5cd3c0679b390R16) and never used

Reply:
lt's used, regarding the duplication: it's on purpose. As it's only used within tests, I think it's less confusing to duplicate this short functionality than to have a dependency between crypto and string